web/modules/custom/dpl_update/dpl_update.install

```diff
@@ -188,6 +188,51 @@ function _dpl_update_add_field_columns(string $field_type, array $fields): strin
   );
 }
 
+/**
+ * Helper function to add a BNF subscription.
+ *
+ * @param string $uuid
+ *   The UUID of the subscription to add.
+ * @param string $label
+ *   The label for the subscription.
+ *
+ * @return string
+ *   Feedback message.
+ */
+function _dpl_update_add_bnf_subscription(string $uuid, string $label): string {
+  $feedback = [];
+
+  if (!\Drupal::moduleHandler()->moduleExists('bnf_client')) {
+    return 'The bnf_client module is not enabled. Subscription could not be created.';
+  }
+
+  $entity_type_manager = DrupalTyped::service(EntityTypeManagerInterface::class, 'entity_type.manager');
+  $subscription_storage = $entity_type_manager->getStorage('bnf_subscription');
+
+  /** @var \Drupal\bnf_client\Entity\Subscription[] $existing */
+  $existing = $subscription_storage->loadByProperties([
+    'subscription_uuid' => $uuid,
+  ]);
+
+  if ($existing) {
+    return "The subscription '$label' ($uuid) already exists. Skipping creation.";
+  }
+
+  $subscription_storage->create([
+    'subscription_uuid' => $uuid,
+    'label' => $label,
+  ])->save();
+
+  $feedback[] = "Successfully created subscription for '$label' ($uuid).";
+
+  $queue_batch = DrupalTyped::service(QueueUIBatchInterface::class, 'queue_ui.batch');
```

Comment:
Why this?

Adding the subscription should trigger an update on the next general cron run.

Explicitly triggering it here slows down the update process and causes it to make external requests to the DT site.

I think we should just add the subscription and let cron do the synchronization at a later time.